### ISSUE TITLE
failed to execute

### ISSUE THREAD

**Issue by warmuuh** [raydac/mvn-jlink#3]
plugin is failing with error: 
```
Caused by: java.lang.IllegalArgumentException: Can't parse file name: bellsoft-jdk13.0.1-src.tar.gz
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider$ReleaseList$Release.<init> (LibericaOpenJdkProvider.java:312)
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider$ReleaseList$Release.<init> (LibericaOpenJdkProvider.java:280)
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider$ReleaseList.<init> (LibericaOpenJdkProvider.java:251)
```

presumably because they added a file to their release (bellsoft-jdk13.0.1-src.tar.gz) which does not ft into the schema. maybe logging a simple warning and continue is good enough?

**Comment by raydac** [comment]
good point, will fix

**Comment by raydac** [comment]
fixed in 1.0.5 maintenance release
